.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+- package-ecosystem: cargo
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  labels:
+  - "\U0001F4E6 dependencies"
```

Comment:
I guess we're gonna need to add `/starship_module_config_derive/` too:
```yaml
- package-ecosystem: cargo
  directory: "/starship_module_config_derive/"
  schedule:
    interval: daily
  open-pull-requests-limit: 10
  labels:
  - "\U0001F4E6 dependencies"
```

Reply:
Right now, CI doesn't touch `starship_module_config_derive`. It is manually deployed and isn't touched by Dependabot. Definitely something worth addressing in a separate PR. 👍

Reply:
> Right now, CI doesn't touch `starship_module_config_derive`.

Except `cargo` lint/test invocations, they use `--workspace` flag IIRC. But yeah, there are surely more things to automate there. :)